docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,75 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. The ratio and difference can be calculated `between_groups` 
+or `to_overall`, but `to_overall` results in more than 1 value being returned.
+:class:`MetricFrame` can also calculate differences and ratios between groups 
+using these methods. For more information on available method of computing 
```

Comment:
The phrasing "using these methods" is a bit confusing for me, as it suggests that `MetricFrame` 'borrows' `'ratio'` and `'difference'` from `equalized_odds_difference` etc. It seems like the parts about `MetricFrame.ratio` and `MetricFrame.difference` are better off in the section about `MetricFrame` itself.